core/kazoo_number_manager/src/knm_phone_number.erl

```diff
@@ -520,7 +520,8 @@ features_list(N) ->
     sets:to_list(sets:from_list(kz_json:get_keys(features(N)))).
 
 -spec set_features(knm_phone_number(), kz_json:object()) -> knm_phone_number().
-set_features(N, Features=?JSON_WRAPPER(_)) ->
+set_features(N, Features) ->
+    'true' = kz_json:is_json_object(Features),
```

Comment:
I believe the macro usage was added to pacify Dialyzer but I could be mis-remembering.